src/pybind/rados/rados.pyx

```diff
@@ -3341,8 +3341,8 @@ returned %d, but should return zero on success." % (self.name, ret))
             int prval = 0
 
         with nogil:
-            rados_read_op_omap_get_vals(_read_op.read_op, _start_after, _filter_prefix,
-                                        _max_return, &iter_addr,  &prval)
+            rados_read_op_omap_get_vals2(_read_op.read_op, _start_after, _filter_prefix,
+                                         _max_return, &iter_addr, NULL, &prval)
```

Comment:
That's the new 'bool more' arg, right? Shouldn't the wrapper be checking that so that the iterator wrapper can reliably report when it has reached the end?